CONTRIBUTING.md

```diff
@@ -1,26 +1,39 @@
 ### Before you Start
-- Unless stated in README.md. Make sure you are pulling the source code from main branch with recursive variable to pull all submodules Collapse need to run.
+- Unless otherwise stated in this project README. Make sure to clone the source code from the `main` branch using `--recurse` parameter to pull all the submodules Collapse need to run.
 
-    ```git pull https://github.com/neon-nyan/Collapse --recursive```
+    ```git pull https://github.com/neon-nyan/Collapse --recurse```
     
 - Make sure you use the supported IDE/SDKs listed below.
-- Make sure to read the Feature(s) restriction below.
-- When its time to do a Pull Request, do a pull request to the same branch when you first pulled.
-- We don't require a fully descriptive PR description, but please be clear on what added/removed and what it does.
+- Make sure to read the "Restriction for New Feature(s)" section below.
+- When submitting a Pull Request, make sure that it is done to the same branch from which you first pulled, unless otherwise stated in the project's README, or if specifically instructed by maintainers of this project.
+- We don't require a fully descriptive PR, but please be clear on what added/removed and what the code does.
 
-### Software Needed
-Below is the software needed to develop and contributing to this project. Also, Collapse uses many submodule inside its repository, so don't forget to use recursive variable when pulling this repository to your local computer.
+### Tools Needed
+Below is the tools needed to contribute to this project.
 1. **Visual Studio 2022 (Any Edition - 17.4 or later)**
 2. .NET: [**.NET Core 7 SDK (7.0.100 or later)**](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
 3. WinUI 3: [**WinUI 3 (WindowsAppSDK 1.3.0-230331000 Stable Runtime)**](https://aka.ms/windowsappsdk/1.3/1.3.230331000/windowsappruntimeinstall-x64.exe)
 4. **Windows 10 SDK (10.0.19043.0 or later)** via Visual Studio Installer

```

Comment:
I think we should only list Windows 10 SDK 10.0.19043.0 and 10.0.22000 since these are the only SDKs supported by Collapse for now

Reply:
Applied